From cbc988374e7fa763d415404660b8fa3ced05b043 Mon Sep 17 00:00:00 2001
From: CraigX Kewley <craigx.kewley@intel.com>
Date: Wed, 22 Mar 2017 12:31:37 +0000
Subject: [PATCH 061/102] ASoC: Intel: Skylake: Increase ROM init retries to
 six

ROM init retries limited to 3, but on Sil.A boards 4/5 are required.

Signed-off-by: CraigX Kewley <craigx.kewley@intel.com>

Conflicts:
	sound/soc/intel/skylake/bxt-sst.c
---
 sound/soc/intel/skylake/bxt-sst.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/intel/skylake/bxt-sst.c b/sound/soc/intel/skylake/bxt-sst.c
index e6b6a17..e97a787 100644
--- a/sound/soc/intel/skylake/bxt-sst.c
+++ b/sound/soc/intel/skylake/bxt-sst.c
@@ -59,7 +59,7 @@
 /* Delay before scheduling D0i3 entry */
 #define BXT_D0I3_DELAY 5000
 
-#define BXT_FW_INIT_RETRY 3
+#define BXT_FW_INIT_RETRY 6
 
 #define GET_SSP_BASE(N)	(N > 4 ? 0x2000 : 0x4000)
 
-- 
1.9.1

